dataprofiler/profilers/profile_builder.py

```diff
@@ -515,7 +516,7 @@ def _update_profile_from_chunk(df, profile=None, sample_size=None,
             raise ValueError('`Profiler` does not currently support data which '
                              'contains columns with duplicate names.')
 
-        for col in df.columns:
+        for col in tqdm(df.columns):
```

Comment:
tqdm shouldn't be a hard requirement IMO. Attempt import, if fails use another method.